src/components/TopicList.jsx

```diff
@@ -71,13 +82,23 @@ const TopicList = () => {
             )}
           </ul>
           <div className="topicList__container__articles">
-            {topicArticles.length > 0 &&
-              topicArticles.map((article) => (
-                <TopicListArticleItem
-                  article={article}
-                  key={article.article_id}
-                />
-              ))}
+            {topicArticles.length > 0 && (
+              <SelectOptions
+                sortBy={sortBy}
+                setSortBy={setSortBy}
+                sortOrder={sortOrder}
+                setSortOrder={setSortOrder}
+              />
+            )}
+            <section className="article__list">
+              {topicArticles.length > 0 &&
+                topicArticles.map((article) => (
+                  <TopicListArticleItem
+                    article={article}
+                    key={article.article_id}
+                  />
```

Comment:
is TopicListItem and ArticleListItem so different that they need to be different components? Could you make one component to be shown in both places?